brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Responsive condition - Updated CSS for hide effect.

### Description

- Update hide effect css.

### Screenshots

- https://share.getcloudapp.com/YEurr6be

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop any block.
- Add a responsive condition & check hide effect.

### Checklist:
